convert-to-json.py

```diff
@@ -15,22 +15,23 @@
 
 system_name = os.path.basename(os.path.realpath(folder_name))
 
-with open(os.path.join(folder_name, "scf.in")) as fhandle:
+with open(os.path.join(folder_name, "GaAs.scf.in")) as fhandle:
     scf_input = fhandle.read()
-with open(os.path.join(folder_name, "scf.out")) as fhandle:
+with open(os.path.join(folder_name, "GaAs.scf.out")) as fhandle:
     scf_output = fhandle.read()
-with open(os.path.join(folder_name, "matdyn.modes")) as fhandle:
+with open(os.path.join(folder_name, "GaAs.modes")) as fhandle:
     matdyn_modes = fhandle.read()
 
-pretty_name_dict = {"BaTiO_3": "BaTiO<sub>3</sub>"}
+pretty_name_dict = {"GaAs": "GaAs"}
```

Comment:
I think this dict should contain entries for all the examples that have a subscript. So GaAs, is not needed, but BaTiO_3 should be kept and everything else that has subscripts should be added (although i'm not sure why it wasn't before). If the entry doesn't exist, i think the rest of the code will just use the correct system name automatically.

Reply:
got it